massgov/mayflower
# DP-16804 State organizations link in Utility Nav

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

Change the state organizations menu from a button to a link directly to the page.

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-16804)

## Steps to Test


1. Go to any page in localhost:3000.
2. Check the State Organizations menu item in the Utility Nav in inspect.
3. Find the menu item is a link. Its href value is "#", not actual URL in Mayflower.
4. Do the step 3 and 4 in mobile display.
5. Pull the branch into Drupal and find the link href value has the URL to the state organizations page.

## Screenshots
Desktop
<img width="815" alt="Pattern_Lab_-_pages-Homepage" src="https://user-images.githubusercontent.com/9633303/72629295-09890c00-391e-11ea-9466-311609963652.png">

Mobile
<img width="502" alt="mobile" src="https://user-images.githubusercontent.com/9633303/72629294-09890c00-391e-11ea-9275-627b11123f0f.png">




## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
